packages/frontend/src/features/omnibar/api/search.ts

```diff
@@ -1,15 +1,24 @@
-import { SearchResults } from '@lightdash/common';
+import { SearchFilters, SearchResults } from '@lightdash/common';
 import { lightdashApi } from '../../../api';
 
 export const getSearchResults = async ({
     projectUuid,
     query,
+    filters,
 }: {
     projectUuid: string;
     query: string;
-}) =>
-    lightdashApi<SearchResults>({
-        url: `/projects/${projectUuid}/search/${encodeURIComponent(query)}`,
+    filters?: SearchFilters;
+}) => {
+    const searchParams = filters
+        ? '?' + new URLSearchParams(filters).toString()
+        : '';
```

Comment:
```suggestion
    const searchParamsString = new URLSearchParams(filters).toString();
    const searchParams = searchParamsString && `?${searchParamsString}`;
```

The current condition will break if `filters` is an empty object. If that ever happens, then searchparams will be `?`

After trying it out, `new URLSearchParams(filters).toString()` will always return `''` if either `filters` is empty object or undefined, so it is safe to call without check and do the check just for the `searchParams` string on the next line

Reply:
fixed